posthog/session_recordings/session_recording_api.py

```diff
@@ -1267,6 +1285,28 @@ def _stream_blob_to_client(
 
                 return response
 
+    async def _stream_lts_blob_v2_to_client_async(
+        self,
+        recording: SessionRecording,
+        timer: ServerTimingsGathered,
+        blob_key: str,
+    ) -> HttpResponse:
+        with STREAM_RESPONSE_TO_CLIENT_HISTOGRAM.labels(blob_version="v2").time():
+            with (
+                timer("list_blocks__stream_lts_blob_v2_to_client_async"),
+                tracer.start_as_current_span("list_blocks__stream_lts_blob_v2_to_client_async"),
+            ):
+                content = await asyncio.to_thread(session_recording_v2_object_storage.client().fetch_file, blob_key)
+
+            twenty_hour_hours_in_seconds = 60 * 60 * 24
```

Comment:
_really_ this could be immutably cached but 24 hours saves us loading this too much if someone is clicking around in one session

Reply:
Typo in the variably name I think - should probably be `twenty_four_hours_in_seconds`